tests/resource/model_resource.py

```diff
@@ -3,97 +3,101 @@
 
 .. moduleauthor:: Peter Morihladko <morihladko@atteq.com>
 """
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *
```

Comment:
In [python-future.org](http://python-future.org/quickstart.html#if-you-are-writing-code-from-scratch) they recommend to use this, so I started to add in every new file.